libs/etterlatte-ktor/src/main/kotlin/route/RouteUtils.kt

```diff
@@ -211,25 +210,6 @@ suspend inline fun PipelineContext<*, ApplicationCall>.withParam(
     }
 }
 
-suspend inline fun PipelineContext<*, ApplicationCall>.hentNavidentFraToken(onSuccess: (navident: String) -> Unit) {
```

Comment:
Denne "skjuler" vel at routsa som bruker den er kun tilgjengelig for saksbehandler og ikke systembruker, om det lenger skal være tilfellet derimot er ikke sikkert.

Reply:
Nja, i #4615 opna eg for systembrukar-kall hit også, så trur ikkje denne PR-en endrar noko på det.